x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/function/Functions.java

```diff
@@ -18,4 +18,12 @@ public static boolean isAggregate(Expression e) {
     public static boolean isGrouping(Expression e) {
         return e instanceof GroupingFunction;
     }
-}
\ No newline at end of file
+
+    public static int countOfNonNullOptionalArgs(Object... args) {
+        int shift = 0;
+        for (Object a : args) {
+            shift += a == null ? 0 : 1;
+        }
+        return shift;
+    }
```

Comment:
This method is not tied to functions, has a clunky name and bad names - countOf -> return shift?.
Please remove it.